Makefile

```diff
@@ -30,6 +30,12 @@ all: rudder-templates-cli.jar
 	java -jar rudder-templates-cli.jar --outext .cf --outdir initial-promises/node-server/inventory/1.0 techniques/system/inventory/1.0/virtualMachines.st
 	java -jar rudder-templates-cli.jar --outext .cf --outdir initial-promises/node-server/inventory/1.0 techniques/system/inventory/1.0/fetchFusionTools.st
 	java -jar rudder-templates-cli.jar --outext .cf --outdir initial-promises/node-server/inventory/1.0 techniques/system/inventory/1.0/fusionAgent.st
+	sed -i -e 's/.*TRACKINGKEY.*/  "TRACKINGKEY": "root-DP@@root-distributePolicy@@00",/' variables.json
+	cp techniques/system/distributePolicy/1.0/rudder-ncf-conf.st initial-promises/node-server/distributePolicy/ncf/ncf.conf
```

Comment:
I'm not comfortable with this just being a simple cp, while the file is a .st file and is listed in the technique's metadata.xml file as a TML (not a file that is just copied).

I realise it doesn't have any StringTemplate syntax in today, but if someone adds some in, this cp will directly include it in initial promises, and probably break *a lot* of things.

I see two alternatives:
1) We change the metadata.xml to list this file as a simple file to copy, not a TML (best, IMHO)
2) We run this file through the jar to de-StringTemplate it for initial-promises, in case one day it contains StringTemplate stuff (unncessary, IMHO, but least requires less changes)